### PR TITLE
How does a draft work?

### PR DESCRIPTION
**Pull request by MicahWW**
I am making this one a draft, maybe we should be making drafts, getting reviews from each other, then make a real pull request, and finally merge.
Let's see.

### CODE REVIEW

**Comment by codyswanner** [comment]
I cannot review it because "this pull request is still a work in progress."  I can click "ready for review" to make it not a draft.

**Comment by codyswanner** [comment]
I do like this, I should have used this feature on #29 and the adding_color branch (deleted), this is exactly what I had in mind for them.  I just didn't realize drafts were a thing

File: TicTacToe.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿import random
 from math import floor
 import os
-
+# blah blah blah
```
Comment (MicahWW) [inline TicTacToe.py:4]: Test comment

**Review by MicahWW — COMMENTED** [review]
You can start a review but can't hit approve.

File: TicTacToe.py (hunk 1 above)
Comment (codyswanner) [reply TicTacToe.py:4]:
```suggestion
# yada yada yada
```

Comment (codyswanner) [reply TicTacToe.py:4]: Code suggestions still work!

**Comment by MicahWW** [comment]
This has been fun but I think we don't need this anymore.

**Pull request closed without merging**